build-system/tasks/e2e/index.js

```diff
@@ -192,7 +192,7 @@ async function runTests_() {
 async function runWatch_() {
   const filesToWatch = argv.files ? getFilesFromArgv() : config.e2eTestPaths;
 
-  log('Watching', cyan(filesToWatch), 'for changes...');
+  log('Watching', cyan(filesToWatch.join(', ')), 'for changes...');
```

Comment:
I believe this is the intended behavior here but the `cyan` function signature does not claim to support a list as an argument.